README.md

```diff
@@ -1,2 +1,200 @@
-# hello-microverse
-First Microverse project to demonstrate effective use of linters, the GitHub flow, and professional documentation
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
[OPTIONAL] Great job implementing the recommended template. Remember that you can always use your own logo or remove it altogether. 😋😋